src/UseCases/ApplyForMembership/ApplyForMembershipRequest.php

```diff
@@ -45,6 +46,15 @@ public function setMembershipType( string $membershipType ) {
 		$this->membershipType = $membershipType;
 	}
 
+	public function getShowMembershipTypeOption(): bool {
```

Comment:
This is presentation specific and (thus) not used by the UC, so should not be in here